src/components/thead/thead-styles.jsx

```diff
@@ -2,23 +2,28 @@ import styleUtils from '../table/style-utilities';
 import color from '../../styles/color';
 
 const normal = theme => {
-  const { palette, mixins, typography } = theme;
+  const { mixins, typography } = theme;
 
   return {
     thead: {
       width: '100%',
       ...styleUtils.sizes(),
       ...mixins.basicBoxSizing,
       fontWeight: typography.fontWeightSemiBold,
-      borderColor: color.gray,
-      borderBottom: `2px solid ${palette.shades.dark.text.muted}`,
+      borderBottom: `2px solid ${color.grayDarker}`,
+      '& th': {
+        '@media screen and (-ms-high-contrast: active), screen and (-ms-high-contrast: none)': {
```

Comment:
the value `none` is no longer supported in IE. What are we trying to solve here?

Reply:
this was just some "if IE" thing I found on google,  we need specific handling to workaround IE bugs and quirks

Reply:
What does it solve? :)